.github/workflows/testing-regular-issues.yml

```diff
@@ -13,46 +13,42 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
 
-#  powershell:
-#    runs-on: windows-latest
-#    steps:
-#      - uses: actions/checkout@v3.1.0
-#      - uses: pCYSl5EDgo/cat@master
-#        id: read_recent
-#        with:
-#          path: sources-for-download/regular-issues.txt
-#      - run: |
-#          echo "ISSUE_NUMBER=${{ steps.read_recent.outputs.text }}" >> $GITHUB_ENV
-#          echo "ISSUE_MINUS_ONE=$((${{ steps.read_recent.outputs.text }}-1))" >> $GITHUB_ENV
-#          echo "ISSUE_MINUS_FIVE=$((${{ steps.read_recent.outputs.text }}-5))" >> $GITHUB_ENV
-#          echo "ISSUE_MINUS_SIX=$((${{ steps.read_recent.outputs.text }}-6))" >> $GITHUB_ENV
-#
-#      - name: Download Issues With end limit
-#        id: normalIssuesWithEndLimit
-#        shell: pwsh
-#        run: ./windows/magpi-issue-downloader.ps1 -l 3
-#
-#      - name: Download Issues with start and end limit
-#        id: normalIssuesWithRange
-#        shell: pwsh
-#        run: ./windows/magpi-issue-downloader.ps1 -f ${{ env.ISSUE_MINUS_SIX }} -l ${{ env.ISSUE_MINUS_FIVE }}
-#
-#      - name: Download Issues with start limit
-#        id: normalIssuesWithStartingLimit
-#        shell: pwsh
-#        run: ./windows/magpi-issue-downloader.ps1 -f $((${{ env.ISSUE_NUMBER }}-1))
-#
-#      - name: "Check file existence Windows"
-#        id: check_files
-#        uses: andstor/file-existence-action@v2.0.0
-#        with:
-#          files: "issues/the-magpi-issue-1.pdf, issues/the-magpi-issue-1.pdf,, issues/the-magpi-issue-${{ env.ISSUE_MINUS_SIX }}.pdf, issues/the-magpi-issue-${{ env.ISSUE_MINUS_FIVE }}.pdf, issues/the-magpi-issue-${{ env.ISSUE_MINUS_ONE }}.pdf, issues/the-magpi-issue-${{ env.ISSUE_NUMBER }}.pdf"
-#          allow_failure: false
-#      - name: lslah issues
-#        run: Get-ChildItem –path D:\a\MagPiDownloader\MagPiDownloader\issues
-#      - name: File exists
-#        if: steps.check_files.outputs.files_exists == 'false'
-#        run: exit 1
+  powershell:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: pCYSl5EDgo/cat@master
+        id: read_recent
+        with:
+          path: sources-for-download/regular-issues.txt
+      - run: |
+          echo "ISSUE_NUMBER=${{ steps.read_recent.outputs.text }}" >> $GITHUB_ENV
```

Comment:
https://github.com/joergi/MagPiDownloader/actions/runs/3425556071/jobs/5745598217
somehow the test is failing.
but I will merch so I can fix this on my GH.